csrc/kernel_ir.h

```diff
@@ -539,6 +539,10 @@ class TORCH_CUDA_CU_API ForLoop final : public Expr {
     return input(0);
   }
 
+  Val* indexOrStart() const {
```

Comment:
Would like to have a better name or at least some comment. I think it would be helpful to mention why this should be used rather than just `index()`.

Reply:
changed to `indexOrStartIfTrivial`